fulp_modules/safety_moth/safety_moth/safety_moth.dm

```diff
@@ -0,0 +1,99 @@
+/datum/ert/safety_moth

```

Comment:
This file isnt in the tgstation.dme, and why is /datum/ert/safety_moth in both files?